README.md

```diff
@@ -288,6 +288,7 @@ Name | Website | Region
 [OmbuLabs](/company-profiles/ombu-labs.md) | https://www.ombulabs.com/ | Worldwide
 [OmniTI](/company-profiles/omniti.md) | https://omniti.com/ | US
 [OpenCraft](/company-profiles/opencraft.md) ⚠️️ | http://opencraft.com/ |
+[OpenSourceWebsite](/company-profiles/opensourcewebsite.md) | https://opensourcewebsite.org | Worldwide
```

Comment:
Hi @grandmotivator, I think that OpenSourceWebsite is not a good fit for this list. Of course we love open-source projects but for this list we are looking for companies who hire remote workers as employees or as freelancers.

Reply:
The organization hires remote employees and freelancers. Welcome.